packages/design-system/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import {
+  pageLink,
+  pageLinkActive,
+  pageLinkWrapper,
+  paginationList,
+} from "./style.css";
+import { compositeClassNames } from "utils/compositeClassNames";
+
+interface PaginationProps {
+  currentPage: number;
```

Comment:
```suggestion
  currentPage?: number;
```
currentPage가 옵셔널이 아닌데 defaultValue를 지정해주신 이유가 있을까요?